tests/constbuffer_ut/constbuffer_ut.c

```diff
@@ -2562,4 +2562,144 @@ TEST_FUNCTION(CONSTBUFFER_from_buffer_with_2_size_unhappy_path)
     ///clean
 }
 
+/*Tests_SRS_CONSTBUFFER_51_001: [ If `size` is 0, then CONSTBUFFER_create_writiable_handle shall fail and return NULL. ]*/

```

Comment:
>CONSTBUFFER_create_writiable_handle [](http://example.com/codeflow?start=55&length=35)

Nitpick: For the naming of this, we usually use the naming convention that the C standard uses (as you wrote it): CONSTBUFFER_create_writiable_handle

But this module that you are touching is veeeeeeery old and has a naming convention like: CONSTBUFFER_CreateWritiableHandle

I would consider using CONSTBUFFER_CreateWritiableHandle so that it is consistent thoughout the module. If we ever rework the module completely then we'd change the convention, but until then it is better to have the module consistent with itself.

Reply:
Renamed all the newly added functions, so that they will be consistent throughout the module.